test/SqlExtension.Tests/TestData.cs

```diff
@@ -1,28 +1,27 @@
-﻿using System;
-
-namespace SqlExtension.Tests
-{
-    #pragma warning disable CS0659 // Type overrides Object.Equals(object o) but does not override Object.GetHashCode()
-    public class TestData
-    #pragma warning restore CS0659
-    {
-        public int ID { get; set; }
-
-        public string Name { get; set; }
-
-        public double Cost { get; set; }
-
-        public DateTime Timestamp { get; set; }
-
-        public override bool Equals(object obj)
-        {
-            var otherData = obj as TestData;
-            if (otherData == null)
-            {
-                return false;
-            }
-            return this.ID == otherData.ID && this.Cost == otherData.Cost && ((this.Name == null && otherData.Name == null) ||
-                string.Equals(this.Name, otherData.Name, StringComparison.OrdinalIgnoreCase)) && this.Timestamp.Equals(otherData.Timestamp);
-        }
-    }
-}
+﻿using System;
+
+namespace SqlExtension.Tests
+{
+    #pragma warning disable CS0659 // Type overrides Object.Equals(object o) but does not override Object.GetHashCode()

```

Comment:
We should be implementing GetHashCode as well https://stackoverflow.com/questions/371328/why-is-it-important-to-override-gethashcode-when-equals-method-is-overridden